prototypes/entity.lua

```diff
@@ -59,16 +60,115 @@ data:extend({
 		{
 			shadow =
 			{
-				red = {2.01, 0.6},
-				green = {2.52, 0.6}
+				red = {2.52, 0.65},
+				green = {2.01, 0.65}
 			},
 			wire =
 			{
-				red = {1.71, 0.3},
-				green = {2.22, 0.3}
+				red = {2.22, 0.32},
+				green = {1.71, 0.32}
 			}
 		},
 	},
+	-- small warehouse
+	{
+		type = "container",
+		name = "warehouse-basic-small",
+		icon = "__Warehousing__/graphics/icons/warehouse-basic-small.png",
+		icon_size = 32,
+		flags = {"placeable-neutral", "placeable-player", "player-creation"},
+		minable = {mining_time = 2, result = "warehouse-basic-small"},
+		max_health = 450,
+		corpse = "big-remnants",
+		dying_explosion = "medium-explosion",
+		open_sound = { filename = "__base__/sound/machine-open.ogg", volume = 0.85 },
+		close_sound = { filename = "__base__/sound/machine-close.ogg", volume = 0.75 },
+		vehicle_impact_sound = { filename = "__base__/sound/car-metal-impact.ogg", volume = 0.65 },
+		resistances =
+		{
+			{
+				type = "fire",
+				percent = 90
+			}
+		},
+		collision_box = {{-2.4, -2.4}, {2.4, 2.4}},
+		selection_box = {{-2.4, -2.4}, {2.4, 2.4}},
+		fast_replaceable_group = "container",
+		inventory_size = warehouse_slots,
+		scale_info_icons = settings.startup["Warehousing-icon-scaling"].value,
+		picture =
+		{
+			filename = "__Warehousing__/graphics/entity/warehouse-basic-small.png",
+			priority = "high",
+			width = 210,
+			height = 190,
+			shift = {0.75, -0.2},
+		},
+		circuit_wire_max_distance = 7.5,
+		circuit_wire_connection_point =
+		{
+			shadow =
+			{
+				red = {2.0, 0.6},
+				green = {1.6, 0.6}
+			},
+			wire =
+			{
+				red = {1.75, 0.3},
+				green = {1.35, 0.3}
+			}
+		},
+	},
+	-- tiny warehouse
+	{
+		type = "container",
+		name = "warehouse-basic-tiny",
+		icon = "__Warehousing__/graphics/icons/warehouse-basic-tiny.png",
+		icon_size = 32,
+		flags = {"placeable-neutral", "placeable-player", "player-creation"},
+		minable = {mining_time = 2, result = "warehouse-basic-tiny"},
+		max_health = 450,
+		corpse = "big-remnants",
+		dying_explosion = "medium-explosion",
+		open_sound = { filename = "__base__/sound/machine-open.ogg", volume = 0.85 },
+		close_sound = { filename = "__base__/sound/machine-close.ogg", volume = 0.75 },
+		vehicle_impact_sound = { filename = "__base__/sound/car-metal-impact.ogg", volume = 0.65 },
+		resistances =
+		{
+			{
+				type = "fire",
+				percent = 90
+			}
+		},
+		collision_box = {{-1.9, -1.9}, {1.9, 1.9}},
```

Comment:
The collision box is meant to be smaller than the selection box. See the `-basic` elements—their collision boxes are 0.3 units smaller than their selection boxes.

Reply:
Discussion will continue on another of these similar changes.